platform.md

```diff
@@ -364,8 +339,8 @@ The platform MUST execute `detector` in the **build environment**
 Usage:
 ```
 /cnb/lifecycle/detector \
-  [-app <app>] \
   [-analyzed <analyzed>] \
+  [-app <app>] \
```

Comment:
We'll need to add `-analyzed` to the builder as well to provide target information during the build phase

Reply:
Added in 1fe9fd4